tests/ballerina-test-integration/src/test/resources/invocationcontext/invocation-context-test.bal

```diff
@@ -0,0 +1,47 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/runtime;
+import ballerina/io;
```

Comment:
Do we need io package here

Reply:
No we don't, fixed with https://github.com/ballerina-platform/ballerina-lang/pull/9071.